pkg/quotes/README.md

```diff
@@ -42,13 +43,333 @@ last_price = price
+```
+**Trade 3 received ```{uniswap, 55000, 0.6}```:** 
+```
+activeWeights(btcusdt) = driverWeights[Binance] + driverWeights[Uniswap] = 2
+tradeSourceWeight = driverWeights[Uniswap] = 0
+sourceMultiplier = tradeSourceWeight/ActiveWeights = 0/2 = 0
+
+NumEMA = EMA20((Volume)*(SourceMultiplier)*(Price))
+DenEMA = EMA20((Volume)*(SourceMultiplier))
+
+<!-- Get previous values from cache -->
+prevNumEMA = priceCache.Get(btcusdt) = 13150.25
+prevDenEMA = priceCache.Get(btcusdt) = 0.319
+
+<!-- EMA calculation -->
+EMA = ((newValue − previous EMA) × smoothingFactor) + (previous EMA).
+
+numEMA = ((55000*0.6*0-13150.25) x 0.095) + 13150.25 = 11900.97625
+denEMA = ((0.6*0-0.319) x 0.095) + 0.319 = 0.288695
+
+priceCache.Update(numEMA, denEMA)
+
+index_price = numEMA/denEMA = 11900.97625/0.288695 = 41223
+```
+
+**Trade 4 received ```{uniswap, 50000, 0.4}```:** 
+```
+sourceMultiplier = tradeSourceWeight/ActiveWeights = 0/2 = 0
+
+prevNumEMA = priceCache.Get(btcusdt) = 11900.97625
+prevDenEMA = priceCache.Get(btcusdt) = 0.288695
+
+<!-- EMA calculation -->
+numEMA = ((50000*0.6*0-11900.97625) x 0.095) + 11900.97625 = 10770.3835063
+denEMA = ((0.4*0-0.288695) x 0.095) + 0.288695 = 0.261268975
+
+priceCache.Update(numEMA, denEMA)
+
+index_price = numEMA/denEMA = 10770.3835063/0.261268975 = 41223
+```
+
+**Trade 5 received ```{binance, 40000, 1.0}```:** 
+```
+activeWeights(btcusdt) = driverWeights[Binance] + driverWeights[Uniswap] = 2
+tradeSourceWeight = driverWeights[Binance] = 2
+sourceMultiplier = tradeSourceWeight/ActiveWeights = 2/2 = 1
+
+prevNumEMA = priceCache.Get(btcusdt) = 10770.3835063
+prevDenEMA = priceCache.Get(btcusdt) = 0.261268975
+
+<!-- EMA calculation -->
+numEMA = ((40000*1.0*1-10770.3835063) x 0.095) + 10770.3835063 = 13547.197
+denEMA = ((1.0*1-0.261268975) x 0.095) + 0.261268975 = 0.332
+
+priceCache.Update(numEMA, denEMA)
+
+index_price = numEMA/denEMA = 13547.1970732/0.33144842237 = 40872
+```
+
+```Weighted EMA price change with each trade: 41000, 41223, 41223, 41223, 40872```
+
+**Example 3:**
+
+- Only Binance driver is active
+- The first 4 trades amount is ```1```
+- The 5-th trade amount is ```10```
+- The price increases by 2000 with each trade:
+
+```
+{source, price, amount}
+{binance, 40000, 1.0}
+{binance, 42000, 1.0}
+{binance, 44000, 1.0}
+{binance, 46000, 1.0}
+{binance, 48000, 10.0}
+```
+
+Calculations flow is the same as in the previous examples, ```sourceMultiplier``` stays at ```1``` because only the Binance driver is active.
+
+```Weighted EMA price change with each trade: 40000, 40190, 40553, 41072, 44624```
+
+The last trade with a larger amount has influenced the final price more significantly than all other trades.
+
+**Example 4:**
+
+- Initial price is 41000
+- Two drivers are active
+- These drivers are sequentially sending 40000 and 42000 trades with the same amount
+
+In this case, the index price will get smoothed, alternating between 41050 and 40950 instead of the initial 40000 and 42000.
```

Comment:
I would not use EMA as in this example we can see the price is always behind the market generating opportunities to milk the system